chat-plugins/trivia.js

```diff
@@ -408,7 +389,14 @@ var Trivia = (function () {
 
 	return {
```

Comment:
Your Trivia object is just a `getInstance` method?

How does this all even work? -_-